rafiki/predictor/ensemble.py

```diff
@@ -54,15 +53,14 @@ def ensemble(predictions: List[Any]) -> Any:
     prediction = _simplify_prediction(prediction)
     return prediction
 
-
 def _simplify_prediction(prediction):
     # Convert numpy arrays to lists
     if isinstance(prediction, np.ndarray):
         prediction = prediction.tolist()
 
-    # Recurvely apply to elements of iterables
-    if isinstance(prediction, Iterable):
+    # Recurvely apply to elements of lists
```

Comment:
would iterable better than list?
I think list is one kind of iterable?

Reply:
This is to avoid the bug mentioned at https://github.com/nginyc/rafiki/issues/141#issuecomment-516296781